lib/utils/invoiceExport/getInvoiceExportData.js

```diff
@@ -81,9 +81,10 @@ export const getInvoiceExportData = async ({ ky, intl, query, currency: to }) =>
   });
   const addresses = getAddresses(addressRecords);
   const currencies = uniq(exportInvoices.map(({ currency }) => currency));
-  const exchangeRates = await Promise.all(
+  const allExchangeRates = await Promise.allSettled(
     currencies.map(from => ky.get(EXCHANGE_RATE_API, { searchParams: { from, to } }).json()),
   );
+  const exchangeRates = allExchangeRates.filter(({ status }) => status === 'fulfilled');
```

Comment:
There is a bug: before changes, `exchangeRates` was an array of exchange rate responses, after - an array of fullfiled promises. I don't think it would work as expected without extracting values